src/YoutubeAtom/YoutubeAtom.tsx

```diff
@@ -81,34 +75,34 @@ export const YoutubeAtom = ({
 }: YoutubeAtomType): JSX.Element => {
     const embedConfig =
         adTargeting && JSON.stringify(buildEmbedConfig(adTargeting));
+
+    // if window is undefined it is because this logic is running on the server side
+    const origin =
+        typeof window !== 'undefined' &&
+        window.location.hostname === 'localhost'
+            ? ''
+            : '&origin=https://www.theguardian.com';
```

Comment:
It sounds like this specifies the origin of the page in which the embed appears. What happens if this is running on AR instead of DCR, where the origin is `https://mobile.guardianapis.com`?

Reply:
this is a good point, another case would be running on `CODE` where the URL will start within something like `https://m.code.dev-theguardian.com`. I just noticed that the JSON for articles returns `host`, so will look at setting that for the origin instead of this logic 👍